### DISCUSSION TITLE
Colorable resizable cusor plz

### DISCUSSION THREAD

**Discussion by Zerq** [linuxmint/discussions#617] [Ideas]
can we have something like the colorable resizable cursor setting windows 10 has 
this is a great aid to accessibility and usability when your on a quad screen setup and to scale the cursor for bigger screen and higher resolutions.

the blip feature would also be nice but being able to make the cursor stand out with a bright color and bigger size probably helps plenty on its own.

This would also align nicely with how the simplified theming view works

**Comment by mtwebster** [comment]
If you're using Cinnamon you can adjust the pointer size in your mouse settings. The color cannot currently be changed, however (apart from black or white).

![image](https://github.com/user-attachments/assets/4ad2fe8e-494a-434c-b8a8-119ea1bff3dd)

**Reply by Zerq** [reply]
ah yeah lol now that you mentionn it i probably did find that but the color option is still something that would be nice and going looking for a theme really is tedious since most are just stupid anime thened novelty cursors half the time or some incompatible cursor theme... its very confusing... i've also had cursor themeing not working in some cases though that may be more a general theming issue...

**Reply by Jopp-gh** [reply]
https://www.gnome-look.org/browse?cat=107&ord=top
